runelite-api/src/main/java/net/runelite/api/AnimationID.java

```diff
@@ -72,8 +72,16 @@
 	public static final int FLETCHING_STRING_YEW_LONGBOW = 6688;
 	public static final int FLETCHING_STRING_MAGIC_SHORTBOW = 6683;
 	public static final int FLETCHING_STRING_MAGIC_LONGBOW = 6689;
-	public static final int FLETCHING_ATTACH_FEATHERS_TO_ARROWSHAFT = 8481;
+	public static final int FLETCHING_ATTACH_BOLT_TIPS_TO_BRONZE_BOLT = 8472;
+	public static final int FLETCHING_ATTACH_BOLT_TIPS_TO_IRON_BOLT = 8473;
```

Comment:
This is the ID used for both iron and broad bolt tip attaching, so update this var's name.
```suggestion
	public static final int FLETCHING_ATTACH_BOLT_TIPS_TO_IRON_BROAD_BOLT = 8473;
```